lms/djangoapps/edxnotes/helpers.py

```diff
@@ -249,11 +218,14 @@ def get_module_context(course, item):
         'location': unicode(item.location),
         'display_name': item.display_name_with_default,
     }
-    if item.category == 'chapter':
+    if item.category == 'chapter' and item.get_parent():
```

Comment:
I know it is cached, but ``item.get_parent()`` is used twice in both paths so should probably be put into a variable.